docs/sources.md

```diff
@@ -103,29 +103,137 @@ builder.Services.AddOpcUaSubjectClient<Sensor>("opc.tcp://localhost:4840", "opc"
 
 Properties with `[SourcePath("opc", "NodeName")]` map to OPC UA nodes in the external server's address space.
 
+## Source Ownership Model
+
+Each property can have **at most one source** at any time. This is the single source of truth model:
+
+- Sources claim ownership of properties by calling `SetSource(this)` during initialization
+- The `SubjectSourceBackgroundService` dispatches changes only to the owning source
```

Comment:
can we just run one background service (instead of per source) and this one just groups by source and calls write?

Reply:
each source add calls TryAddSingleton?

Reply:
too complicated for now

Reply:
to complex